starcheck/calc_ccd_temps.py

```diff
@@ -119,6 +120,12 @@ def get_ccd_temps(oflsdir, outdir='out',
 
     :returns: JSON dictionary of labeled dwell intervals with max temperatures
     """
+    # For kadi commands v2 running on HEAD set the default scenario to flight.
+    # This is aimed at running in production where the commands archive is
+    # updated hourly. In this case no network resources are used.
+    if test_helper.on_head_network():
+        os.environ.setdefault('KADI_SCENARIO', 'flight')
```

Comment:
What would be the downside to setting the default KADI_SCENARIO to flight in general here?

Reply:
Then you can't run starcheck on a laptop without remembering to `ska_sync` the commands archive h5/pkl files.

Also you can't use the kadi commands scenario machinery to run a non-standard scenario. Granted that might never happen for ACA, but it's there and there is no good reason starcheck can't work with that.

Reply:
I haven't played much with setdefault but I figured with a name like "default" that it could be overridden by the user for non-standard scenario.   Or is there no good place to do that?

Seems like perhaps we should also have some checking here about if the cmds archive is new enough.  We've got that checking for the v1 archive as part of ska jobwatch (and probably need to add it for v2?) but for the load review tool maybe a sanity check would be good too.

Reply:
We have two common situations:

1. Running on HEAD. The "flight" archive is guaranteed to be up to date and we have cron checkers that ensure this is the case. We don't want a separate check in starcheck because we can't really know the right answer. If chandra has been in safe mode for a week then there won't be recent commands. All we can do is check that the job to update commands has run recently.
2. Running on a laptop. The *raison d'etre* of commands v2 is to get away from needing to sync the commands archive. We want the DEFAULT to be for kadi commands to go out and fetch commands using network resources to guarantee being up to date.

Reply:
> for the load review tool maybe a sanity check would be good too.

If you really want to push on this then it's OK but that is a separate issue unrelated to this PR to do the transition to v2 since it also applies to current flight starcheck using v1.

Reply:
Good point on there's no right answer for when the commands have been updated.  With regard to the two situations, I think I understand this and yes for the running on a laptop use case, I figured the user could set the environment variable or use a cmd line option or we could make a different wrapper.  I suppose I'm not 100% forward-thinking on this issue about "running ska_sync" being a burden because we always want the updated agasc supplement and this code isn't configured to use maude yet either, so the user has to have an up-to-date cheta archive anyway.  Overall I'm onboard with making starcheck more mobile, just not sure in this case if the default setting needs to be conditional.